graphblas/backend/cuda/spgemm.hpp

```diff
@@ -108,6 +110,182 @@ Info spgemmMasked(SparseMatrix<c>*       C,
   C->csc_initialized_ = false;
   return GrB_SUCCESS;
 }
+// Shallow copy graphblast sparsematrix -> Galatic dCSR format
```

Comment:
nit: please add a newline between following template function and above function.